h/presenters/group_json.py

```diff
@@ -10,27 +10,42 @@ def __init__(self, group, links_svc=None):
         self.group = group
         self._links_svc = links_svc
 
-    def asdict(self):
-        return self._model(self.group)
+    def asdict(self, expand=[]):
+        model = self._model()
+        self._expand(model, expand)
+        self._inject_urls(model)
+        return model
+
+    def _expand(self, model, expand=[]):
```

Comment:
This first implementation is inelegant, if functional. In a subsequent PR, I'd like to create an `OrganizationJSONPresenter` and shove the logic in there. Didn't want this PR to get too huge.

Reply:
...and potentially abstract the notion of `expand` into something reusable. But didn't want to prematurely over-architect.